modules/labs-react/common/lib/utils/position.ts

```diff
@@ -0,0 +1,95 @@
+import {
+  ContentDirection,
+  PartialEmotionCanvasTheme,
+  useTheme,
+} from '@workday/canvas-kit-react/common';
+import {Property} from 'csstype';
+
+/** style props to for standard position properties */
+export type PositionStandardProps = {
+  /** sets `position` property  */
+  position?: Property.Position;
+  /** sets `zIndex` property  */
+  zIndex?: Property.ZIndex;
+  /** sets `top` property  */
+  top?: number | string;
+  /** sets `right` property (no bidirectional support) */
+  right?: number | string;
+  /** sets `bottom` property  */
+  bottom?: number | string;
+  /** sets `left` property (no bidirectional support)  */
+  left?: number | string;
+};
+
+export type PositionLogicalProps = {
+  /** sets `left` property (bidirectional support)  */
+  insetInlineStart?: number | string;
```

Comment:
Is this standard naming? The `inset` confused me for a minute

Reply:
Yes: https://css-tricks.com/css-logical-properties/

Reply:
Agreed that it's initially confusing. The JSDoc hint + docs (+ maybe a lint rule in the future?) should help with that.